sympy/physics/optics/medium.py

```diff
@@ -120,9 +120,15 @@ def speed(self):
         >>> m = Medium('m')
         >>> m.speed
         299792458*meter/second
+        >>> m2 = Medium('m2', n=1)
+        >>> m.speed == m2.speed
+        True
 
         """
-        return 1/sqrt(self._permittivity*self._permeability)
+        if self._permittivity != None and self._permeability != None:
```

Comment:
Not sure why I didn't think about it earlier, but it is in general better to write `is not None` than `!= None`.

Reply:
Thank you, done.